cap-ci/pipeline.yaml.tmpl

```diff
@@ -73,8 +73,8 @@ resources:
   type: s3
   source:
     bucket: {{ $config.s3minibroker.bucket }}
-    access_key_id: ((aws-cf-user-suse-rd.access-key))
-    secret_access_key: ((aws-cf-user-suse-rd.secret-key))
+    access_key_id: ((aws-access-key))
```

Comment:
should it not be `aws-capbot-access-key` ?

Reply:
There seems to be a mismatch between secret names in concourse.suse.de and concourse.suse.dev. One way to fix it was to do the change here and sawp aws-access-key and aws-capbot-access-key values in concourse.suse.dev.

With the latest developments, seems we need to change the secrets layout in Vault so the Concourse instances can seamlessly pull them.

Reply:
I think its confusing to name rd account creds as `aws-access-key/secret-key`. In black box we have a liberty to add comments but we don't have the same in k8s secrets.
I am okay with using `aws-capbot-access-key/secret-key`, but for rd account I guess we should stick to `aws-cf-user-suse-rd.access-key/secret-key`, for clarity.
Also, we have this issue right now with rd account https://jira.suse.com/browse/CAP-1657

Reply:
I agree, the name is confusing, `aws-cf-user-suse-rd.access-key/secret-key` is a better name. But migrating all the pipelines and reflying them to consume it is not trivial (sometimes one has problems when flying some pipelines).
This would get solved automatically with https://github.com/cloudfoundry-incubator/kubecf/issues/1607 or https://github.com/cloudfoundry-incubator/kubecf/issues/1608, but maybe we just need to fix it apart from those cards.

Reply:
These 2 latest commits (yet to be merged) may inform on what the secrets need to be:
https://github.com/SUSE/cloudfoundry/pull/213/commits/9a24ef5dc9578159a642d5febd2d23209b86c885
https://github.com/SUSE/cloudfoundry/pull/213/commits/e12187619fe028a8d4822c8c60640c572797796b